WD-Firmware/examples/build/SConscript_bsp_nexys_a7_swerv1

```diff
@@ -0,0 +1,74 @@
+'''
+ SPDX-License-Identifier: Apache-2.0
+ Copyright 2019 Western Digital Corporation or its affiliates.
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http:www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+'''
+import os
+import utils
+Import('Env')
+
+out_dir = os.path.join(Env['OUT_DIR_PATH'], 'bsp')
+utils.createFolder(out_dir)
+    
+bsp_base=os.path.join('board', 'nexys_a7', 'bsp')
+
+#list_files_c=[
+#   [os.path.join(bsp_base, 'env', 'freedom-e300-hifive1', 'init.c'), os.path.join(out_dir, 'init.o')],
+#   [os.path.join(bsp_base, 'drivers', 'plic', 'plic_driver.c'), os.path.join(out_dir, 'plic_driver.o')],
+#]
+
+list_files_c=[]
+    
+list_files_a=[

```

Comment:
add comments for _a and _c on all filels

Reply:
done